packages/block-library/src/post-excerpt/block.json

```diff
@@ -31,5 +31,6 @@
 		},
 		"lineHeight": true
 	},
-	"editorStyle": "wp-block-post-excerpt-editor"
+	"editorStyle": "wp-block-post-excerpt-editor",
+	"style": "wp-block-post-excerpt"
```

Comment:
What is this and why is a change necessary on the front-end?

Reply:
This is to match editor with front-end styles.